### DISCUSSION TITLE
Warning: The post-install step did not complete successfully

### DISCUSSION THREAD

**Discussion by razan-rai** [shivammathur/homebrew-php#1987] [Q&A / Help] [answered]
This is the warning i got while installing php@7.4  more details of the error output is here.

dyld[79929]: Library not loaded: '/usr/local/opt/libavif/lib/libavif.15.dylib'
  Referenced from: '/usr/local/Cellar/gd/2.3.3_5/lib/libgd.3.dylib'
  Reason: tried: '/usr/local/opt/libavif/lib/libavif.15.dylib' (no such file), '/usr/local/lib/libavif.15.dylib' (no such file), '/usr/lib/libavif.15.dylib' (no such file), '/usr/local/Cellar/libavif/1.0.1/lib/libavif.15.dylib' (no such file), '/usr/local/lib/libavif.15.dylib' (no such file), '/usr/lib/libavif.15.dylib' (no such file)

**Comment by shivammathur** [comment, marked as answer]
@razan-rai 

Please run

```zsh
brew upgrade gd
brew post-install shivammathur/php/php@7.4
```

**Reply by razan-rai** [reply]
Thank you, Actually `brew post-install shivammathur/php/php@7.4` got and error
`Error: Unknown command: post-install` but after updating my shell profile ~/.bash_profile with `export PATH="/usr/local/opt/php@7.4/bin:$PATH"` iam able to run php@7.4

**Reply by shivammathur** [reply]
It should be `brew postinstall shivammathur/php/php@7.4` without the `-`.

In case you want to switch to other versions, please run the postinstall command and remove php@7.4's bin from the path, or else it will always run php 7.4.

**Reply by razan-rai** [reply]
@shivammathur Once again much appreciated.
